config/services.yaml

```
@@ -0,0 +1,11 @@
+services:
+    Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigExtension:
+        tags:
+            - { name: twig.extension }
+
+    Oneup\ContaoSentryBundle\Twig\ContaoSentryTwigRuntime:
+        public: false
+        tags:
+            - { name: twig.runtime }
+
+    Oneup\ContaoSentryBundle\Integration\IgnorePreviousExceptionsIntegration: ~
```

As of current best practices we should not use FQCN as service ids.
```suggestion
    oneup_contao_sentry.integration.ignore_previous_exceptions:
        class: Oneup\ContaoSentryBundle\Integration\IgnorePreviousExceptionsIntegration
```